src/reference/faq.rst

```diff
@@ -70,24 +68,49 @@ The other deciding factor is if you're unable to install Docker (e.g. for admins
 The ambient runtime is not available without WSL.
 
 
-.. _why-intel-miniconda-installer-on-apple-silicon:
+.. _why-conda-install-errors-on-apple-silicon:
+
+Why do I get an error when installing to my Conda environment on a Mac computer with `Apple silicon`_ (e.g. M1)?
+----------------------------------------------------------------------------------------------------------------
+
+.. _Apple silicon: https://support.apple.com/en-us/HT211814
+
+An example error:
+
+.. code-block:: text
+
+   (my-conda-env) $ mamba install augur
```

Comment:
s/my-conda-env/your-environment-name/ so it matches the rest of our docs?

Reply:
Updated in [force-push](https://github.com/nextstrain/docs.nextstrain.org/compare/a651379456c5ff1e8cf35eebaa7b19fbd73c8adf..675e98db2f8e06feb21ae690edad37249fe33d89).